Makefile

```diff
@@ -27,7 +27,7 @@ clean:
 .PHONY: deps
 deps: clean
 	( cd ./queue/testdata; $(TTCTL) rocks install queue 1.3.0 )
-	( cd ./crud/testdata; $(TTCTL) rocks install crud 1.1.1 )
+	( cd ./crud/testdata; $(TTCTL) rocks install vshard 0.1.24; $(TTCTL) rocks install crud 1.1.1 )
```

Comment:
Why do we need the change?

Reply:
Since r563 SDK provides tt, tt is used instead of tarantoolctl to install rocks. tt 1.0.2 provided by SDK fails to build a latest vshard rock [1] (the issue is fixed in tt 1.1.1), so we must explicitly require the pipeline to install tagged vshard for now.

1. https://github.com/tarantool/tt/commit/12bf40453614a89a4c898c2ad8e2da739e6de0af

Reply:
Well, it seems that new SDK had been published yesterday, so I may use this one without this workaround.

Reply:
Migrated to new SDK, removed this workaround